recipes/numpy/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+cat > site.cfg <<'siteconfig'
+[DEFAULT]
+library_dirs = $PREFIX/lib
+include_dirs = $PREFIX/include
+
+[atlas]
+atlas_libs = openblas
+libraries = openblas
+
+[openblas]
+libraries = openblas
+library_dirs = $PREFIX/lib
+include_dirs = $PREFIX/include
+siteconfig
```

Comment:
In past experience, I have found doing this linking trick to be simpler and easier to maintain. I don't know if it carries over to Windows though.